WikiStuff.py

```diff
@@ -0,0 +1,123 @@
+from bs4 import BeautifulSoup
+from sphinx.util import requests
+
+HEADERS_GET = {
+    'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; WOW64; rv:49.0) Gecko/20100101 Firefox/49.0',
+    # 'User-Agent': 'Mozilla/5.0 (Linux; Android 7.0; SAMSUNG SM-G950F Build/NRD90M) AppleWebKit/537.36 (KHTML, like '
+    #             'Gecko) SamsungBrowser/5.2 Chrome/51.0.2704.106 Mobile Safari/537.36',
+    'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8',
+    'Accept-Language': 'en-US,en;q=0.5',
+    'Accept-Encoding': 'gzip, deflate',
+    'DNT': '1',
+    'Connection': 'keep-alive',
+    'Upgrade-Insecure-Requests': '1'
+}
+
+WIKI_SEARCH_RESULT_ATTRS = {"class": "mw-search-result-heading"}
+WIKI_SEARCH_RESULT_DESCRIPTION_ATTRS = {"class": "searchresult"}
+WIKI_TABLE_ATTRS1 = {"style": "background-color:#fff"}
+WIKI_TABLE_ATTRS2 = {"style": "background-color:#f7f7f7"}
+
+
+def is_number(n):
+    try:
+        float(n)  # Type-casting the string to `float`.
+        # If string is not a valid `float`,
+        # it'll raise `ValueError` exception
+    except ValueError:
+        return False
+    return True
+
+
+def findWikiUrl(album, artist):
+    found = False
+    query = "{title} {art}".format(art=artist, title=album).replace(" ", "+").replace("&", "and")
+    search = "https://en.wikipedia.org/w/index.php?cirrusUserTesting=glent_m0&sort=relevance&search={query}&title=Special:Search&profile=advanced&fulltext=1&advancedSearch-current=%7B%7D&ns0=1".format(
+        query=query)
+    res = requests.get(search, headers=HEADERS_GET).text
+    soup = BeautifulSoup(res, 'html.parser')
+    tags = [tag for tag in soup.find_all(attrs=WIKI_SEARCH_RESULT_ATTRS)]
+    items_hrefs = ['https://en.wikipedia.org/{}'.format(tag.find_next('a', href=True).get('href')) for tag in tags]
+    print("item_hrefs= {}".format(items_hrefs))
+    for j in items_hrefs:
```

Comment:
this loop will almost always return the first result (if it has the album in  it its enough)